test/cli/package-no-spec/__package.rb

```diff
@@ -0,0 +1,2 @@
+# typed: strict
+# This package doesn't have a packagespec defined. It will error.
```

Comment:
I see you have a mix of `test/cli/` tests and `test/testdata/` tests.

Are all the `test/cli/` tests ones that can't be written as `test/testdata/` tests? Those are usually far easier to work with than cli tests, so I'm curious to learn about what the reasons are for the `test/cli/` tests existing.

(Understandably we probably want at least one test to make sure that the `--stripe-packages` flag works when turned on / off, but it looks to me like there are more tests than just that, so I'm wondering what the goal is with them.)

Reply:
> Are all the test/cli/ tests ones that can't be written as test/testdata/ tests?

That was the idea. One tests a corner case with `--stripe-mode` and `--stripe-packages` that `test/testdata` can't exercise, and a few others involved errors that I couldn't assert bc they happened on the `typed:` sigil or their locs covered multiple lines.

I'll double check that this is still true.

Reply:
I reduced the CLI tests down to 2.